ia/src/ml_agent.py

```
@@ -45,114 +48,119 @@ def search_closest_resource(client, needed_resources):
                 break
 
     return closest_resource
-      
+
+
 def get_action(client, needed_resources):
```

Ruff (0.11.9)

`get_action` is too complex (11 > 10)

(C901)